src/bpm/config/job_config.go

```diff
@@ -64,7 +64,7 @@ type Unsafe struct {
 	Privileged bool `yaml:"privileged"`
 }
 
-func ParseJobConfig(jobName, configPath string) (*JobConfig, error) {
+func ParseJobConfig(configPath string) (*JobConfig, error) {
```

Comment:
This is the bit that I started not to like when trying this. It'll be easy to make the mistake of using this to get a JobConfig and then not validating it.